Formula/qt-gstreamer.rb

```diff
@@ -0,0 +1,52 @@
+class QtGstreamer < Formula
+  desc "C++ bindings for GStreamer with a Qt-style API"
+  homepage "https://gstreamer.freedesktop.org/modules/qt-gstreamer.html"
+  url "https://gstreamer.freedesktop.org/src/qt-gstreamer/qt-gstreamer-1.2.0.tar.xz"
+  sha256 "9f3b492b74cad9be918e4c4db96df48dab9c012f2ae5667f438b64a4d92e8fd4"
+  head "https://anongit.freedesktop.org/git/gstreamer/qt-gstreamer.git"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+
+  depends_on "boost"
+  depends_on "glib"
+  depends_on "gstreamer"
+  depends_on "gst-plugins-base"
+  depends_on "qt5"
+
+  def install
+    args = std_cmake_args
+    args << "-DQT_VERSION=5"
+    args << "-DUSE_GST_PLUGIN_DIR=OFF"
+    args << "-DUSE_QT_PLUGIN_DIR=OFF"
+    args << "-DCMAKE_CXX_FLAGS=-I#{Formula["gstreamer"].opt_lib}/gstreamer-1.0/include"
+    system "cmake", *args
```

Comment:
Since there's no variability here (all the arguments are always passed) you can save some quoting and repetition by simply concatenating two arrays like this:

```ruby
args = std_cmake_args + %W[
  -DQT_VERSION=5
  -DUSE_GST_PLUGIN_DIR=OFF
  -DUSE_QT_PLUGIN_DIR=OFF
  -DCMAKE_CXX_FLAGS=-I#{Formula["gstreamer"].opt_lib}/gstreamer-1.0/include
]
system "cmake", *args
```

(But the current style is also fine.)